### ISSUE TITLE
alt text feature

### ISSUE THREAD

**Issue by Fleaurent** [mushanshitiancai/vscode-paste-image#128]
Markdownlint always complains when i paste an image via vscode-paste image:
"MD045/no-alt-text: Images should have alternate text (alt text) markdownlint[MD045]"(https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md045.md)
Could you provide a feature also generating an automated alt text i.e. the entered image name?
